script/data-harmonizer/export_utils.js

```diff
@@ -163,8 +164,8 @@ Object.assign(DataHarmonizer, {
 	 */
 	getTransformedField: function (headerName, value, field, prefix) {
 
-	 	if (field['schema:ItemList']) {
-	 		const term = this.findById(field['schema:ItemList'], value);
+		if (field.sources) {
+			term = DataHarmonizer.schema.enums[field.sources].permissible_values[value];		
```

Comment:
Hmm - field.sources is an array of enum names (keys).  I think a loop has to occur on field.sources to look at each enum key's permissable_values